projects/material-addons/src/lib/quick-list/quick-list.component.html

```diff
@@ -5,7 +5,7 @@ <h3>
   <div *ngIf="itemTemplate" #row>
     <ng-container *ngTemplateOutlet="itemTemplate; context: { $implicit: item }"> </ng-container>
     <button *ngIf="!(readonly || readonly === '')" mat-icon-button [disabled]="!isDeleteAllowed()" (click)="removeItem(item)">
-      <mat-icon>delete</mat-icon>
+      <mat-icon color="primary">delete_outline</mat-icon>
```

Comment:
In master-branch this change was already done by replacing it with the new <mad-icon>